docker/config/docker_common.env

```diff
@@ -109,5 +111,5 @@ RABBITMQ_VIRTUAL_HOST=rabbitvhost
 RAVEN_DSN=
 SECRET_KEY=secretkey
 SESSION_COOKIE_SECURE=False
-SYMBOLS_BUCKET_DEFAULT_NAME=
+SYMBOLS_BUCKET_DEFAULT_NAME=symbols
 SYMBOLS_BUCKET_EXCEPTIONS=
```

Comment:
The changes in this file are specific to our local dev environment--nothing fancy here.